src/components/Home.vue

```diff
@@ -78,7 +78,7 @@ export default {
   mounted() {
     this.$kuzzle.on('tokenExpired', () => this.onTokenExpired())
     this.$kuzzle.on('queryError', e => {
-      if (e.code === 401) {
+      if (e.id === "0x07010001") {
```

Comment:
Using `security.token.invalid` doesn't work?

Reply:
Yes @morgandruesne, hexadecimal formats are not intend to be used on "high level application" but only when error size (in bytes) is a prerequisite for low level applications (like embedded software maxed to 32 bits in average)

Reply:
I'm gonna do this ! thanks for the explanation :smile: